pkg/kubelet/kubelet.go

```diff
@@ -1554,6 +1554,10 @@ func (kl *Kubelet) syncLoop(updates <-chan PodUpdate, handler SyncHandler) {
 			unsyncedPod = true
 		case <-time.After(kl.resyncInterval):
 			glog.V(4).Infof("Periodic sync")
+		case <-time.After(3 * kl.resyncInterval):
```

Comment:
I think this will never happen - the above condition (waiting for kl.resyncInterval above) will always happen before it.